secant/Util/SeedIndex.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  SeedIndex.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 17.02.2022.
+//
+
+@propertyWrapper
+struct SeedIndex {
```

Comment:
Currently `@Clamped` is only being used on Enumerated chip. So, for this PR to be complete. the Clamped property wrapper should be deleted since it would be no longer in use.

Reply:
let's stick with the Clamped but check the detailed comment for the 2nd comment